modules/venat-module-market/README.md

```diff
@@ -0,0 +1 @@
+# venat-module-market
```

Comment:
IMO we probably want to name XIV-related modules like `venat-module-xiv-market` so the distinction is clearer once possibly non-XIV game modules are added.

Reply:
Updated